src/components/Common/LoginAndJoin/Components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import '../Components/Header.scss';
```

Comment:
동일 경로에 있기 때문에 다음처럼 접근하시면 됩니다.
```suggestion
import React from 'react';
import './Header.scss';
```